docker-compose.yml

```diff
@@ -10,6 +10,7 @@ services:
       - 8000:8000
     env_file:
       - ./.env.dev
+    container_name: people_depot_dev
```

Comment:
Same reasoning as above for not setting the container name. We're mainly referring to containers using `docker-compose`.